spyglass.py

```diff
@@ -437,6 +437,10 @@ def sanitize(string: str) -> Optional[str]:
 
 logger.info("Spreadsheet populated!")
 print("Saving spreadsheet...")
+
+outfile_path = Path(args.outfile)
+outfile_path.parent.mkdir(parents=True, exist_ok=True)
```

Comment:
I am concerned about the following bit from the docs:
> If exist_ok is true, [FileExistsError](https://docs.python.org/3/library/exceptions.html#FileExistsError) exceptions will be ignored (same behavior as the POSIX mkdir -p command), but only if the last path component **is not an existing non-directory file.**

From my reading of this, this means that if you don't pass a new filename (irrespective of the folder), it will raise `FileExistsError`. Could you please wrap this in a `try/except` block?